16_woman_cancer.do

```diff
@@ -11,8 +11,11 @@ if _rc==0 {
 	replace w_papsmear=. if s714dd==9 | s714ee==9
 }
 
-
 capture confirm variable s1017 s1020 
 if _rc==0 {
     replace w_mammogram=. if s1017==. | s1017==9 | s1020==9
 }
+
+// I'm wodering where do you find these codes? 
+// Be careful these two variables may be country specific in recode six too.
+// Also may need to add them in the quality control file, as they could be compared with HEFPI database.
```

Comment:
Please do and update in the Github for this indicator in the Quality control code.